apps/judicial-system/web/src/components/InfoCard/InfoCardClosedIndictment.tsx

```diff
@@ -40,6 +41,9 @@ const InfoCardClosedIndictment: FC<Props> = (props) => {
     (log) => log.eventType === EventType.INDICTMENT_REVIEWED,
   )?.created
 
+  const showMergeCaseItem =
+    isNonEmptyArray(mergeCase.values) && !!mergeCase.values[0]
```

Comment:
There are now two ways to merge a case with another. 

1. Merge it with a case that is in RVG. In this case, the id of the case in RVG is set in the `mergeCaseId` field in DB. 
2. Merge it with a case that is not in RVG. In this case, the court case number is set in the `mergeCaseNumber` field in db.

To check if we should show the merge case id we'd need to check if either of these fields are set, i.e.

`
const showMergeCaseItem = workingCase.mergeCase || workingCase.mergeCaseNumber
`

Reply:
As discussed, we ensure to check this in the hook where `mergeCase` `Item` is configured to avoid double checking if a case has internal or external case merge. But I will move this check to the hook since I think this is simply a utility to decide weather to render an item or not which is useful for other info cards 👍